book/forms.md

```diff
@@ -827,6 +868,21 @@ Try it! You should be able to restart the server, open it in your
 browser, and update the guest book a few times. You should also be
 able to use the guest book from a real web browser.
 
+::: {.further}
```

Comment:
Does it make sense to call out specific technologies? It feels a bit like advertising, since there are many other options besides these.

Reply:
I removed the mentions of Flask and Django. It maybe feels appropriate to mention bottle, since we use it?

Reply:
sgtm